### PR TITLE
Fix minor issues with the Dataverse transfer type

### PR DESCRIPTION
**Pull request by ross-spencer**
:construction: :construction: :construction: 

Connected to archivematica/issues#269.

### CODE REVIEW

File: src/dashboard/src/main/migrations/0061_create_dataverse_transfer_type.py [hunk 1]
```diff
@@ -131,6 +131,22 @@
...
+SET_TRANSFER_TYPE_DESC = "Set transfer type: Dataverse"
+TRANSFER_IN_PROGRESS_DESC = "Set Dataverse transfer in progress"
+SET_CONVERT_DATAVERSE_DESC = "Set convert Dataverse structure flag"
+SET_PARSE_DATAVERSE_DESC = "Set parse Dataverse METS flag"
+CONVERT_DATAVERSE_DESC = "Convert Dataverse structure"
+DETERMINE_CONVERSION_DESC = "Determine if Dataverse conversion needs to happen"
+DETERMINE_PARSE_DATAVERSE_DESC = \
+    "Determine if Dataverse METS XML needs to be parsed"
+PARSE_DATAVERSE_METS_DESC = "Parse Dataverse METS XML"
+
```
Comment (sevein) [inline src/dashboard/src/main/migrations/0061_create_dataverse_transfer_type.py:149]: Can't comment above but there's a small typo:
`EXISTING_MS_VAILIDATE_FORMATS`

**Review by sevein — COMMENTED** [review]
@ross-spencer does this need to be rebased?

**Review by sevein — CHANGES_REQUESTED** [review]
Needs to be rebased.

**Comment by ross-spencer** [comment]
Closed in favor of https://github.com/artefactual/archivematica/pull/1332

**Pull request closed without merging**